From eaa5a4f8a9ea4e8ee448538faa798933c41870c9 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Wed, 4 Dec 2019 22:47:25 +0100
Subject: t7415: drop v2.20.x-specific work-around

This reverts the work-around that was introduced just for the v2.20.x
release train in "t7415: adjust test for dubiously-nested submodule
gitdirs for v2.20.x"; It is not necessary for v2.21.x.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
(cherry picked from commit d9061ed9dacb5657bcff4260fc35a831946907ec)
Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
---
 t/t7415-submodule-names.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/t7415-submodule-names.sh b/t/t7415-submodule-names.sh
index 0f040e30a4..905a557585 100755
--- a/t/t7415-submodule-names.sh
+++ b/t/t7415-submodule-names.sh
@@ -244,7 +244,7 @@ test_expect_success 'git dirs of sibling submodules must not be nested' '
 		git commit -m nested
 	) &&
 	test_must_fail git clone --recurse-submodules nested clone 2>err &&
-	test_i18ngrep -E "(is inside git dir|hippo already exists|not a git repository: .*/hippo)" err
+	test_i18ngrep "is inside git dir" err
 '
 
 test_done
-- 
2.24.0.393.g34dc348eaf

